k8s/pkg/registryserver/registry_cache.go

```diff
@@ -46,6 +47,7 @@ func NewRegistryCache(clientset *nsmClientset.Clientset) RegistryCache {
 		clientset:                   clientset,
 		stopFuncs:                   make([]func(), 0, 3),
 		nsmNamespace:                namespace.GetNamespace(),
+		createOrUpdate:              sync.Mutex{},
```

Comment:
No need to create it. It is already defined at line 39

Reply:
oops! Thanks, fixed